File: Server/IBigDataPortal/Articles.Domain/ArticlesAggregate/ViewModels/ArticlesVm.cs
using EduLinks.Contracts.ViewModels;
using JobOffers.Domain.PostsAggregate.ViewModels;
using Posts.Domain.PostsAggregate.ViewModels;

namespace Articles.Domain.ArticlesAggregate.ViewModels;

public class ArticlesVm
{
    public IEnumerable<EduLinkViewModel> EduLinks { get; set; }
    public IEnumerable<PostViewModel> Posts { get; set; }
    public IEnumerable<JobOfferViewModel> JobOffers { get; set; }
}

File: App/src/interfaces/Models/Articles/ViewModels/ArticlesVm.ts
import { EduLinkViewModel } from "interfaces/Models/EduLinks/ViewModels/EduLinkViewModel";
import { JobOfferViewModel } from "interfaces/Models/JobOffers/ViewModels/JobOfferViewModel";
import { PostViewModel } from "interfaces/Models/Posts/ViewModels/PostViewModel";

export interface ArticlesVm {
  eduLinks: EduLinkViewModel;
  jobOffers: JobOfferViewModel;
  posts: PostViewModel;
}
